src/psyclone/psyir/nodes/node.py

```diff
@@ -807,11 +809,13 @@ def view(self, indent=0, index=None):
         '''
         from psyclone.psyir.nodes import Schedule
         if not isinstance(self.parent, Schedule) or index is None:
-            print("{0}{1}".format(self.indent(indent),
-                                  self.node_str(colour=True)))
+            result = "{0}{1}".format(self.indent(indent),
+                                     self.node_str(colour=True))
+            print(six.text_type(result))
         else:
-            print("{0}{1}: {2}".format(self.indent(indent), index,
-                                       self.node_str(colour=True)))
+            result = "{0}{1}: {2}".format(self.indent(indent), index,
+                                          self.node_str(colour=True))
+            print(six.text_type(result))
```

Comment:
Please bring this `print` out of the `if` block so that you can remove the one at line 814.

Reply:
Oops.